Issue 150 Slider Mode

nguyenhongphat0 opened the pull request:
Slider mode is now different for each avatar and player
Resolves #150

helenvarley commented:
it is still the same for all media on the stage for me as one player - when i changed the slider function on one avatar, the other avatar and prop on the stage are also changed. 
i'm alone on the stage so can't test at the moment if it has also changed for other players or not.

nguyenhongphat0 commented:
> it is still the same for all media on the stage for me as one player - when i changed the slider function on one avatar, the other avatar and prop on the stage are also changed.
> i'm alone on the stage so can't test at the moment if it has also changed for other players or not.

These changes was not approved and rolled to app1 at the moment you test it, sorry Helen 😅 I just make a build, you can test it now
